wgs/wgs-gather-vcfs/wgs-gather-vcfs.wdl

```diff
@@ -129,8 +129,8 @@ task gather_vcfs_cloud {
     # Find the specified chromosome, find the start and end lines in the intervals file.
     # The file must list chromosome intervals together.
     # Line numbers are 1-indexed, while the intervals file is 0-indexed, so we must subtract 1.
-    START=$(($(grep -n "chr${chromosome}:" "${intervals_file}"| cut -d':' -f1 | head -1) - 1))
-    END=$(($(grep -n "chr${chromosome}:" "${intervals_file}"  | cut -d':' -f1 | tail -1) - 1))
+    START=$(echo $(($(grep -n "chr${chromosome}:" "${intervals_file}"| cut -d':' -f1 | head -1) - 1)))
+    END=$(echo $(($(grep -n "chr${chromosome}:" "${intervals_file}"  | cut -d':' -f1 | tail -1) - 1)))
```

Comment:
Interestingly, another way to fix this is just by using the "readonly" or "declare" statements here:

```
readonly START=$(($(grep -n "chr${chromosome}:" "${intervals_file}"| cut -d':' -f1 | head -1) - 1))
```

as "readonly" and "declare" are statements themselves with their own exit status. It looks like they are the last commands in the pipeline.

I mention this because it would not be bad to make those updates anyway:

```
readonly START=...
readonly END=...

declare INPUT_PARAMETERS=""

...

readonly COMMAND=
```